### DISCUSSION TITLE
Documentation for specific version

### DISCUSSION THREAD

**Discussion by izaacdb** [prisma/prisma#12675] [Q&A] [answered]
Hello - Is it possible to read documentation for a specific version of Prisma? I need to use `"prisma": "2.30.3"` due to some problems introduced by version 3.

**Comment by janpio** [comment, marked as answer]
No, besides going back in the prisma/docs history to that state this is unfortunately not possible.

All added functionality should be annotated with the version it was added though, so the current docs should work for all versions.

>  I need to use "prisma": "2.30.3" due to some problems introduced by version 3.

Is there a bug report for these problems?

**Reply by izaacdb** [reply]
Thanks for this info it is helpful.

I don't think it is worth your time to fix it. v3 can't handle the seed file I have built for v2, but v2 works just fine.

**Reply by Jolg42** [reply]
I think you would benefit to upgrade 👀 So I'm curious:
Is there anything more specific about seeding not working for you in v3? Is it about the change introduced to `prisma db seed` or maybe something in your custom script?

The release notes for that version [are here](https://github.com/prisma/prisma/releases/tag/3.0.1)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
